### DISCUSSION TITLE
Uncaught ReferenceError: Bun is not defined

### DISCUSSION THREAD

**Discussion by Tuymurodov** [oven-sh/bun#10198] [General]
I was using Bun to read values from env in my project I made in react. But when I run the project I get this error. The project was written in typescript so I also installed **@types/bun** but it didn't solve the problem.
![image](https://github.com/oven-sh/bun/assets/92261936/4c8f8bdb-7ac2-4422-8a4c-a31939891bd5)

I wrote the following code to read the values in env: 
`Bun.env.VITE_API_URL`

**Comment by Tuymurodov** [comment]
can anyone help?

**Reply by truongan07** [reply]
try : console.log(globalThis.Bun);

**Reply by Tuymurodov** [reply]
Uncaught TypeError: Cannot read properties of undefined (reading 'env')

now the following error occurred 😒
